agent-bridge/src/main/java/com/newrelic/agent/bridge/Instrumentation.java

```diff
@@ -59,6 +58,10 @@ ExitTracer createTracer(Object invocationTarget, int signatureId, boolean dispat
 
     ExitTracer createScalaTxnTracer();
 
+    default ExitTracer createTracer(String metricName, int flags) {
```

Comment:
The `createSegment` API still generates an asynchronous tracer, which is not what we want.  And from the bridge there's no easy way to register a signature to get a `signatureId`.  This was the easiest way to get a normal, synchronous tracer.